operator/controllers/btpoperator_controller.go

```diff
@@ -978,39 +1151,44 @@ func (r *BtpOperatorReconciler) deleteAllOfinstalledResources(ctx context.Contex
 	return nil
 }
 
-func (r *BtpOperatorReconciler) HandleReadyState(ctx context.Context, cr *v1alpha1.BtpOperator) error {
+func (r *BtpOperatorReconciler) HandleReadyState(ctx context.Context, cr *v1alpha1.BtpOperator) (bool, error) {
```

Comment:
method signature change is not required as it should only be called when CR is in Ready state